protocol/0042-LIQF-setting_fees_and_rewarding_lps.md

```diff
@@ -12,6 +12,10 @@ The aim of this specification is to set out how fees on Vega are set based on co
 
 ## Calculating liquidity fee factor
 
+There are three ways to set the liquidity fee factor, and the default method is the 'Ascending Clock Auction'.
+
+###  Ascending Clock Auction method 
```

Comment:
I don't think this is the correct name for what we have. https://en.wikipedia.org/wiki/Japanese_auction

Reply:
I think it's just the "marginal" method.